main.go

```diff
@@ -120,6 +120,7 @@ func main() {
 		}
 
 		filename := fmt.Sprintf("config-split-%d", i)
+		filename = cont.Name + ".yaml"
```

Comment:
This seems like a good idea, but we don't need to set the name twice.
You could delete the previous line.